mesheryctl/cmd/start.go

```diff
@@ -43,14 +45,68 @@ var startCmd = &cobra.Command{
 
 		fmt.Println("Starting Meshery...")
 		start := exec.Command("docker-compose", "-f", dockerComposeFile, "up", "-d")
+		time.Sleep(2 * time.Second)
+		//testing purpose
+		//exec.Command("docker", "kill", "meshery_meshery_1")
 		start.Stdout = &out
 		start.Stderr = &stderr
 
 		if err := start.Run(); err != nil {
 			fmt.Println(stderr.String())
 			return
 		}
+		// Name of all running container
+		cmda := exec.Command("docker", "ps", "--format", "{{.Names}}")
+		outa, err := cmda.Output()
+		s := strings.Split(string(outa), "\n")
+		//fmt.Println(s)
+		if err != nil {
+			log.Fatalf("cmd.Run() failed with %s\n", err)
+		}
+		check(s, 0)
+	},
+}
+
+func check(s []string, atempt int) {
+	count := 0
+	for _, x := range s {
+		if strings.TrimRight(x, " ") == "meshery_meshery_1" {
+			count++
+		}
+	}
+	if count < 1 && atempt < 1 {
+		//checks for meshery_meshery_1 container running or not if not then restarting meshery
+		fmt.Println("service cannot be started, attempting to restart the service!")
+		exec.Command("mesheryctl", "stop")
```

Comment:
When invoking another `mesheryctl` command, can we do so by calling the Cobra command / function directly? (e.g. `stopCmd.Run()`)?

Reply:
okay Understood